modules/installation-methods-for-different-architectures.adoc

```diff
@@ -6,25 +6,18 @@
 = Installation methods for different architectures
 
 [role="_abstract"]
-{rh-rhacs-first} supports the following architectures.
+{rh-rhacs-first} supports the following architectures. For information on supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix].
 
-.Supported architectures and recommended installation methods
+.Architectures and recommended installation methods
 [%autowidth]
 |===
-|*Supported architectures*|*Supported for Central*|*Supported for Secured Clusters*|*Supported installation methods*
+|*Supported architectures*|*Supported installation methods*
 
 |AMD64
-|Yes
-|Yes
 a|Operator (preferred), Helm charts, or `roxctl` CLI (not recommended)
 
 | ppc64le ({ibm-power})
-|Yes
-|Yes ({ocp} version 4.12 and later)
 .2+a|Operator (the only supported install method)
```

Comment:
Beyond scope of this but:

```suggestion
.2+a|Operator (the only supported installation method)
```

Alternatively, I wonder if you could just list "Operator" and leave out the "the only supported installation method" part.